packages/config/config/devices/0x0258/nas-rp01z1.json

```
@@ -0,0 +1,144 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd",
+	"manufacturerId": "0x0258",
+	"label": "NAS-RP01Z1",
+	"description": "Neo Coolcam Repeater",
+	"devices": [
+		{
+			"productType": "0x0010",
+			"productId": "0x0716"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Low Temperature Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "High (?) Temperature Basic Set",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Low Humidity Basic Set",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "High (?) Humidity Basic Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Red LED blink when measuring",
+			"defaultValue": 1
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Yellow LED blink during OTA",
+			"defaultValue": 1
+		},
+		{
+			"#": "3",
+			"label": "Temperature Offset Value",
+			"description": "[Value] * 0.1 Degree Celsius / Fahrenheit (US)",
+			"valueSize": 1,
+			"minValue": -120,
+			"maxValue": 120,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "4",
+			"label": "Humidity Offset Value",
+			"description": "[Value] * 0.1 RH%",
```

dito - please apply to all similar occurences
```suggestion
			"unit": "0.1 %rH",
```

---

done